acqa-repo1-aws-tf12-part1.tf

```diff
@@ -325,21 +340,21 @@ resource "aws_kms_key" "acqa-test-kmskey1" {
   description             = "acqa-test-kmskey1"
   deletion_window_in_days = 30
   tags = {
-    Name = format("%s-kmskey1", var.acqaPrefix)
+    Name         = format("%s-kmskey1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "ACQA"
+    Owner        = "ACQA"
   }
 }
 
 # ebs volume
 resource "aws_ebs_volume" "acqa-test-ebsvolume1" {
   availability_zone = "ca-central-1a"
   size              = 25
-  encrypted         = false
+  encrypted         = true
   tags = {
-    Name = format("%s-ebsvolume1", var.acqaPrefix)
+    Name         = format("%s-ebsvolume1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "ACQA"
+    Owner        = "ACQA"
   }
 }
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure virtual private cloud (VPC) is configured for AWS EC2 instances**
 
**Vulnerability**
By default AWS EC2 instances are launched in default VPC which is not recommended for production grade. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations.
 
**Remediation**
Migrate from EC2 classic to VPCs. 1: Create a VPC. 2: Configure Your Security Group. 3: Create an AMI from Your EC2-Classic Instance. 4: Launch an Instance Into Your VPC.

source: accurics-